tensorflow/core/ops/math_ops.cc

```diff
@@ -732,6 +732,17 @@ The polygamma function is defined as:
 where \\(\psi(x)\\) is the digamma function.
 )doc");
 
+REGISTER_OP("Atan2")
+    .Input("x1: T")
+    .Input("x2: T")
+    .Output("z: T")
+    .Attr("T: {float, double}")
+    .SetShapeFn(shape_inference::BroadcastBinaryOpShapeFn)
+    .Doc(R"doc(
+Computes arctangent of x1/x2 element-wise, respecting signs of the arguments.
+This is the angle of x2 + x1 * i.
+)doc");
```

Comment:
After looking at the numpy docs and shivering, we decided it'll be best to stick with the time tested atan2(y,x) convention. Could you revise the docstring and arg names?

For reference: 
Python's math.atan2: https://docs.python.org/2/library/math.html
Numpy's arctan2: https://docs.scipy.org/doc/numpy/reference/generated/numpy.arctan2.html